admin-cli/executor/client.go

```diff
@@ -45,14 +46,18 @@ type Client struct {
 }
 
 // NewClient creates a client for accessing Pegasus cluster for use of admin-cli.
-func NewClient(writer io.Writer, metaAddrs []string) *Client {
+// When listing nodes fails, willExit == true means call os.Exit().
+func NewClient(writer io.Writer, metaAddrs []string, willExit bool) (*Client, error) {
```

Comment:
When `willExit` would be set `false` ? Is it necessary to add the param ?

And returned error is also be ignored, is it needed ?

Reply:
The context is that I am estimating the required disk space for a bulk load on the Pegasus gateway using admincli, but currently, admincli does not support disconnect tcp from the replica server. 
When using admincli to connect to the Pegasus cluster, it should not call the os.Exit function; it should only return an error.